plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEditorUtil.kt

```diff
@@ -106,16 +109,15 @@ object CodeWhispererEditorUtil {
     }
 
     /**
-     * Checks if the language is json or yaml and checks if left context contains keywords
+     * Checks if the language is json and checks if left context contains keywords
      */
-    fun checkLeftContextKeywordsForJsonAndYaml(leftContext: String, language: String): Boolean = (
-        (language == CodeWhispererJson.INSTANCE.languageId) ||
-            (language == CodeWhispererYaml.INSTANCE.languageId)
-        ) &&
-        (
-            (!CodeWhispererConstants.AWSTemplateKeyWordsRegex.containsMatchIn(leftContext)) &&
-                (!CodeWhispererConstants.AWSTemplateCaseInsensitiveKeyWordsRegex.containsMatchIn(leftContext.lowercase(Locale.getDefault())))
-            )
+    fun isConfigFileIfJsonFile(fileName: String, leftContext: String, language: CodeWhispererProgrammingLanguage): Boolean =
```

Comment:
`isConfigFileIfJsonFile` method name a bit confusing

Reply:
makes sense, will rename it to `isJsonFormatSupported` maybe?

Reply:
isJsonFileSupported

Reply:
i did this instead `isSupportedJsonFormat`